app/src/main/java/org/sil/gatherwords/EntryActivity.java

```diff
@@ -218,9 +223,39 @@ protected void onPostExecute(List<Long> wordIDs) {
         }
     }
 
+    private static class StorePicturesTask extends AsyncTask<Bitmap, Void, Void> {
 
-    // Camera/Image processing
-    // based on https://developer.android.com/training/camera/photobasics.html
+        private final AppDatabase db;
+        private final WordDao wordDao;
+        private final long wordId;
+
+        StorePicturesTask(EntryActivity activity) {
+            db = AppDatabase.get(activity.getApplicationContext());
+            wordDao = db.wordDao();
+            EntryFragment fragment = (EntryFragment)((EntryPagerAdapter) activity.pager.getAdapter()).getItem(activity.pager.getCurrentItem());
```

Comment:
Seems like a candidate to pull into a method.  More usable if we could call something like `activity.getCurrrentEntryFragment()`, since it sounds like we will be reusing this code.